### ISSUE TITLE
Not compatible with Shopware 5.2.12?

### ISSUE THREAD

**Issue by metachris** [shopwareLabs/SwagBrowserLanguage#6]
Hi,

Is this plugin compatible with Shopware 5.2.12? It doesn't ever redirect me to a language shop (en is the main shop, de is the language shop).

Any advice?

Thanks

**Comment by teiling88** [comment]
Hi, 

what kind of language is active in your Browser? The redirect is based on it and the correct plugin configuration.

**Comment by metachris** [comment]
Hi,

I've used a Firefox extension to set the `Accept-Language` HTTP header to "de_DE", which matches the locale of the language shop.

Should that work, or is there another way to test whether the plugin works?

Thanks

**Comment by teiling88** [comment]
Hi, 
i tested it without a plugin:
![image](https://cloud.githubusercontent.com/assets/4624237/20758083/fbb13394-b718-11e6-9183-e4589767eb50.png)

Important you must configure the shops correct in the plugin config.

**Comment by metachris** [comment]
It doesn't work for me. I've got a main shop with Localization `English (United Kingdom)` and a language shop with localization `German (Germany)`. I've tried changing the Firefox `accept_language` headers as in your screenshot. The headers do arrive at the shopware Apache as `accept-language: de-DE,de;q=0.5`.

This is my plugin config:
![screen shot 2016-12-05 at 11 12 32](https://cloud.githubusercontent.com/assets/116939/20881257/d237bf80-badb-11e6-8c8c-a9c3ff708475.png)
![screen shot 2016-12-05 at 11 12 43](https://cloud.githubusercontent.com/assets/116939/20881258/d23abc4e-badb-11e6-8435-8a170964f5ca.png)

Any idea what I can try?
